test/src/cfile/test_cfile.cpp

```diff
@@ -0,0 +1,29 @@
+
+#include <gtest/gtest.h>
+
+#include <cfile/cfile.h>
+
+#include "util/FSTestFixture.h"
+
+class CFileTest : public test::FSTestFixture {
+ public:
+	CFileTest() : test::FSTestFixture(0) {
+		pushModDir("cfile");
```

Comment:
I don't understand the hows and whys, but there seems to be something wrong with the ways directories are handled in these tests. Changing this line to pushModDir("cfile/temp_subdir") makes the temp_subdir tests work, and changing it to pushModDir("cfile/temp_subdir_vps") makes the temp_subdir_vps tests work. Added debugging statements say that all file lookup happens relative to test_data/cfile/, not relative to test_data/cfile/temp_subdir/ and test_data/cfile/temp_subdir_vps/ as it should.

However, I don't know why for example adding a test like ASSERT_TRUE(cf_exists_full("test.eff", CF_TYPE_ANY)) doesn't cause a(nother) failure. Debug output tells me that it looks in the wrong path (as described above), yet that doesn't cause the test to fail.